src/components/draw/DrawDirective.js

```diff
@@ -37,12 +38,116 @@ goog.require('ga_map_service');
 
         if (useTemporaryLayer) {
           dfltLayer.displayInLayerManager = false;
+          dfltLayer.preview = true;
         }
-
+        gaMeasure.registerOverlaysEvents(map, dfltLayer);
         dfltLayer.label = 'Drawing';
         dfltLayer.type = 'KML';
         return dfltLayer;
       };
+      // Find the corresponding style
+      var findIcon = function(olIcon, icons) {
+        var id = olIcon.getSrc();
+        for (var i = 0; i < icons.length; i++) {
+          var regex = new RegExp('/' + icons[i].id + '-24');
```

Comment:
It's just the convention name of the icons, 24 it's the size in pixel. Inthe previous version I changed the size of the icons dynamically.